### DISCUSSION TITLE
Script tool menu not showing

### DISCUSSION THREAD

**Discussion by drcesimpson** [zsviczian/obsidian-excalidraw-plugin#1036] [Q&A]
My scripts were showing on the screen under a hamburger menu with all the icons when I first started using Excalidraw about a week ago, but they have disappeared. The downloaded scripts are under the script/download folder on the left of the screen. Is there a toggle or some way to have the icons return? Screenshot is from a YT video to show what I am now missing. I have searched here, online and in the documentation for an answer.
![Screenshot 2023-02-25 at 23-49-16 Obsidian-Excalidraw 1 8 10 - Quality of Life improvements](https://user-images.githubusercontent.com/126121887/221391802-aef33b45-a6ba-42bc-8b78-9ab6bf5ad252.png)

**Comment by bingling00** [comment]
I also encountered this problem

**Reply by bingling00** [reply]
I restarted obsidian and solved the problem

**Reply by obsidianbible** [reply]
Same issue with me. Have reopened Obsidian multiple times. The scripts showed up initially, but when reopening the drawing they are not there.

**Reply by kampiamodikuesto** [reply]
I have same problem, all script seems installed but on "obsidian panel" there aren't no script buttons. i try also to reinstall but nothing seems works.

**Reply by ramarivera** [reply]
Ensure the scripts folder in settings is properly upper/lower cased 😅 
This solved my issue https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/1607
